src/main/java/com/alvarium/Sdk.java

```diff
@@ -33,8 +34,8 @@ public interface Sdk {
    * @throws AnnotatorException
    * @throws StreamException
    */
-  public void create(PropertyBag properties, byte[] data) 
-      throws AnnotatorException, StreamException;
+  public void create(PropertyBag properties, byte[] data)
+      throws AnnotatorException, StreamException, HashTypeException;
```

Comment:
I am seeing some errors regarding SDK tests, probably due to the test methods not throwing this exception type.

Reply:
mmh, mine works, are you doing another set of tests that are not in the source code ?